compiler/src/dmd/dsymbolsem.d

```diff
@@ -2217,7 +2218,7 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
 
         if (ed.members.length == 0)
         {
-            ed.error("enum `%s` must have at least one member", ed.toChars());
+            .error(ed.loc, "%s `%s enum `%s` must have at least one member", ed.kind, ed.toPrettyChars, ed.toChars());
```

Comment:
This introduced a diagnostic regression - there's an odd number of backticks present in the format string.

Reply:
https://github.com/dlang/dmd/pull/17002

Reply:
Codecov says this isn't covered by any file in the testsuite either.